benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -2028,6 +2029,7 @@
     "PostConditionState": "*",
     "EventEnablingCondition": "appeal.benefitType.code=\"childSupport\"",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_TRIBUNALS_API_URL}/ccdAboutToSubmit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_SSCS_CCD_ORCHESTRATOR_URL}/send",
```

Comment:
Just wondering, can we handle this in tribunal and then send confirmPanelComposition if only needed? updateOtherPartyData has to send otherPartyAdded event to notification as well. I think tribunal is the best place to handle both in one place.

Reply:
if we put in Tribunals you won't be able to send a notification Rasika, hence the reason this needs to be pointed at Orchestrator - do you agree?

Reply:
yeah, but what I am doing is sending a new event to orchestrator